### ISSUE TITLE
Retrying request

### ISSUE THREAD

**Issue by ramiyahya** [Flibber/Flibber#14]
I keep getting a retrying request error. I've made sure all my options are correct. What am I missing?

 PS - The client secret below is not my real client secret.

``` python
[NUM#] Request #1
[RURL] https://api.instagram.com/oauth/access_token
[FLDS] code=25ec47e1e5ea46fea893cd628f6763b8&client_id=d00d52fb28b4471ebc0b7e43058f5511&redirect_uri=http%3A%2F%2Framiyahya.com&access_token=&client_secret=0d90y591f5d4653a76421ef34fcfcce&grant_type=authorization_code
[HTTP] POST
[CODE] 200
[ACST] Here is your ACCESS_TOKEN: 39040519.d00d52f.0218ceb5661a4755ac50102976f305ce
[REPL] Replace the value in options.py
[TIME] Paused for 60 seconds...
[NUM#] Request #2
[RURL] https://api.instagram.com/v1/users/self/follows
[FLDS] access_token=39040519.d00d52f.0218ceb5661a4755ac50102976f305ce&client_id=d00d52fb28b4471ebc0b7e43058f5511
[HTTP] GET
[RTRY] Retrying request...
[NUM#] Request #3
[RURL] https://api.instagram.com/v1/users/self/follows
[FLDS] access_token=39040519.d00d52f.0218ceb5661a4755ac50102976f305ce&client_id=d00d52fb28b4471ebc0b7e43058f5511
[HTTP] GET
[RTRY] Retrying request...
[NUM#] Request #4
[RURL] https://api.instagram.com/v1/users/self/follows
[FLDS] access_token=39040519.d00d52f.0218ceb5661a4755ac50102976f305ce&client_id=d00d52fb28b4471ebc0b7e43058f5511
[HTTP] GET
[RTRY] Retrying request...
```

**Comment by erenemre** [comment]
> [ACST] Here is your ACCESS_TOKEN: 39040519.d00d52f.0218ceb5661a4755ac50102976f305ce
> [REPL] Replace the value in options.py

Have you updated your access_token variable?

**Comment by ramiyahya** [comment]
Yes I have updated my access token.

On Tuesday, April 28, 2015, Eren Emre Kanal notifications@github.com
wrote:

> [ACST] Here is your ACCESS_TOKEN:
> 39040519.d00d52f.0218ceb5661a4755ac50102976f305ce
> [REPL] Replace the value in options.py
> 
> Have you updated your access_token variable?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Flibber/Flibber/issues/14#issuecomment-96724971.

## 

Rami Yahya

www.ramiyahya.com
0430 457 475

**Comment by chrisbradleyrun** [comment]
Also getting the exact same behaviour here. Any thoughts? Thanks

**Comment by sudo3d** [comment]
I was also experiencing this error and managed to get around it by clearing cookies, deleting the authorized application, and then recreating the application with the "Disable implicit OAuth" and "Enforce signed requests" security options ticked off.

**Comment by hassanshaikley** [comment]
Same.

**Comment by hassanshaikley** [comment]
I unchecked the two options and it works now.

![screen shot 2015-05-06 at 2 46 49 am](https://cloud.githubusercontent.com/assets/6746840/7491025/2af94554-f39a-11e4-9a15-db815bac0d92.png)

I didn't write this library, in fact I just downloaded it, but I'll try to look into this next time that I get the chance. Signed is definitely better, and we definitely want those boxes checked. 

*EDIT: \* Doesn't seem to actually work. I looked at the posts I have liked on IG (trying to get the bot to like) and it shows up empty.
